pkg/haproxy/renderer.go

```diff
@@ -20,7 +20,8 @@ func RenderConfig(data TemplateData) (string, error) {
 		log.Error(err)
```

Comment:
Do we not print this err at the caller? If not, log there. If yes, remove logging here.